common/translations/lang/en.json

```diff
@@ -110,6 +110,7 @@
     "ADD_LABEL_7": "Add Account ",
     "ADD_LABEL_8": "Password (optional): ",
     "ADD_LABEL_9": "add label",
+    "MNEMONIC_SALT_PASSWORD": "This unique password field is also known as \"Salt\", and is an additional (but optional) layer of protection. If you did not set up this additional password when your seed phrase was generated, you do not need to enter anything into this field.",
```

Comment:
Is it known as "salt" though? I think most people, at least in the Ethereum ecosystem, refer to it as "25th word".

Reply:
> Many wallets provide an option to encrypt your mnemonic phrase with a password. This is useful as accessing your wallets would require you to have something you have (the mnemonic phrase) plus something you know (the password). Often times this is referred to as 13th/25th word/seed.

https://support.mycrypto.com/general-knowledge/cryptography/how-do-mnemonic-phrases-work

I copied the copy that was provided on the CH ticket, but you're right, we should make it consistant with our own support article